boardsite-io/boardsite
# Refactor API sessions ♻️

## Summary
This PR restructures the logic inside `api` for better maintainable code:
* created `Session` class to wrap the session management
* removed session logic from `session` redux reducer
* created `Request` class for handling direct API request

## Testing Steps
1. Create an online session
2. Verify drawing/editing behavior in session, e.g. adding/deleting pages, adding PDFs...
